README.md

```diff
@@ -270,8 +270,9 @@ rsconnect write-manifest notebook my-notebook.ipynb
 You can deploy a variety of APIs and applications using sub-commands of the
 `rsconnect deploy` command.
 
-* `api`: WSGI-compliant APIs such as Flask and packages based on Flask
-* `fastapi`: ASGI-compliant APIs (FastAPI, Quart, Sanic, and Falcon)
+* `api`: WSGI-compliant APIs (e.g., `gunicorn`, `werkzeug`, `uwsgi`, `pyramid`, and `bottle`)
```

Comment:
`gunicorn` and `uwsgi` I think are WSGI hosts, vs. frameworks for apps/APIs that can be deployed.

We have tested `bottle`, `flask-api`, `flask-restx`, and `flasgger`, but I don't think we have validated Pyramid.

There content we have integration tests for would be a good source of info here.

Reply:
I don't know of pyramid tests, either.

On the runtime side, we definitely check:

* python-api: simple WSGI (no framework), bottle, falcon, pycnic
* python-dash: dash
* python-streamlit: streamlit
* python-bokeh: bokeh
* python-fastapi: fastapi, quart, sanic, starlette
* python-shiny: shiny

I may be missing some; @kgartland-rstudio has additional test coverage to the ones I'm looking at.

Reply:
Oy. I just realized that my set of runtime tests doesn't include flask! 😭 I'll get that updated.

Reply:
It seems that we should have explicit commands for each of the frameworks that we actively test. 

Until this week, I didn't know that `fastapi` supported other frameworks which implement the ASGI. I assume there are others like me.

This is outside of the scope of this issue, so we can have the conversation here, but I won't implement those changes here.